x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
+    no_fail_fast: bool,
+    #[structopt(long)]
+    /// Do not run tests, only compile the test executables.
+    no_run: bool,
+
+    #[structopt(long, parse(from_os_str))]
+    /// Setting an hmtlcovdir will result in grcov being run for the workspace, or crate the
+    /// command is executed from.   A clean will be performed before running the tests.
+    htmlcovdir: Option<OsString>,
+
     #[structopt(name = "TESTNAME", parse(from_os_str))]
+    /// Setting an hmtlcovdir will result in grcov being run for the workspace, or crate the
+    /// command is executed from.   A clean will be performed before running the tests.
     testname: Option<OsString>,
+
     #[structopt(name = "ARGS", parse(from_os_str), last = true)]
     args: Vec<OsString>,
 }
 
 pub fn run(mut args: Args, config: Config) -> Result<()> {
     args.args.extend(args.testname.clone());
 
-    let cmd = CargoCommand::Test(&args.args);
+    let env_vars: &[(&str, &str)] = if args.htmlcovdir.is_some() {
+        info!("Since we are running coverage, we must run cargo clean first");
```

Comment:
```suggestion
        info!("Running `cargo clean` before collecting coverage");
```

Reply:
Used  "Running \"cargo clean\" before collecting coverage"

Several log line right after this use quotes heavily.    Less visually jarring.    You are quite the wordsmith.